src/main/java/io/ticktok/Ticktok.java

```diff
@@ -12,9 +12,9 @@ public Ticktok(TicktokOptions options){
         this.options = options;
     }
 
-    public void newClock(String schedule, Runnable runnable) {
+    public void newClock(String schedule, String name, Runnable runnable) {
```

Comment:
name should be first and I think it's too many params 
maybe something like this: newClock(name).invoke(func).on(schedule)

Reply:
fixed